addons/dialogue_tree/scripts/dialogue.gd

```diff
@@ -117,6 +119,8 @@ func send_conditonal_data(dict):
 		if current_node.has("Choices"):
 			var ChoicesNew = current_node.Choices
 			for i in ChoicesNew:
+				i["Dialogue"] = i.FormattedDialogue.format(dict)
+				i["Tooltip"] = i.Dialogue.format(dict)
```

Comment:
this line needs to change to:
`i["ToolTip"] = i.ToolTip.format(dict)`
for tool tips formatting to work